app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,86 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ */
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "comments";
+@import "pictures";
+
+body {
+  background-color: rgb(250, 250, 250);
+}
+
+.navbar {
+  background-color: rgb(260, 260, 260);
+  text-align: center;
+}
+
+.navbar-header {
+  margin-bottom: 20px;
+}
+
+.panel-default {
+  position: relative;
+}
+
+article img {
+  width: 100%;
+}
+
+.panel {
+  padding-top: 20px;
+}
+
+.image-container {
+  text-align: center;
+  display: block;
+  width: auto;
+}
+
+.upload {
+  text-align: center;
+}
+
+.user-manage {
+  display: inline-block;
+  margin-left: 10px;
+  float: right;
+}
+
+.alert-error {
+  background-color: #f2dede;
+  border-color: #eed3d7;
+  color: #b94a48;
+  text-align: left;
+}
+
+.alert-alert {
+  background-color: #f2dede;
+  border-color: #eed3d7;
+  color: #b94a48;
+  text-align: left;
+}
+
+.alert-success {
+  background-color: #dff0d8;
+  border-color: #d6e9c6;
+  color: #468847;
+  text-align: left;
+}
+
+.alert-notice {
+  background-color: #dff0d8;
+  border-color: #d6e9c6;
```

Comment:
Color literals like `#d6e9c6` should only be used in variable declarations; they should be referred to via variable everywhere else.